lib/search/global/projects-content.js

```diff
@@ -21,6 +23,8 @@ module.exports = client => async (term = '', query = {}) => {
     fragment_size: 250,
     fields: {
       'content.*': {},
+      'grantedContent.*': {},
+      'ntsContent.*': {},
```

Comment:
Is there any benefit to making these conditional? Since my understanding is that a match within each is not mutually exclusive, are we going to get the same content highlighted in multiple fields?

Reply:
I wasn't really sure what this part did so just copied the content part :derp: